edx_exams/apps/core/migrations/0021_add_courseexamconfiguration_escalation_email.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.21 on 2023-12-04 21:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0020_auto_20231010_1442'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='historicalexam',
+            options={'get_latest_by': 'history_date', 'ordering': ('-history_date', '-history_id'), 'verbose_name': 'historical exam'},
+        ),
+        migrations.AlterModelOptions(
+            name='historicalexamattempt',
+            options={'get_latest_by': 'history_date', 'ordering': ('-history_date', '-history_id'), 'verbose_name': 'historical exam attempt'},
+        ),
```

Comment:
huh something does seem off here. Also looking into what might cause this

Reply:
I think the jacked up migration might be because of the previous migration and this [pull request](https://github.com/edx/edx-exams/pull/185). Note [this comment](https://github.com/edx/edx-exams/pull/185/files#diff-bc839bae89bc2c2671a44f242e95332f02a732b77c37dbcae299d36433170ff3R23).

I wonder if I generated this pull request on a branch off a stale version of master. I'll try recreating the file and see if that fixes it.

Reply:
Hm. It's smaller (no changes to exam attempt tables), but it still modifies the exam table.

```
# Generated by Django 3.2.23 on 2023-12-06 20:56

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0020_auto_20231010_1442'),
    ]

    operations = [
        migrations.AddField(
            model_name='courseexamconfiguration',
            name='escalation_email',
            field=models.EmailField(blank=True, max_length=254),
        ),
        migrations.AlterField(
            model_name='exam',
            name='exam_type',
            field=models.CharField(choices=[('onboarding', 'onboarding'), ('practice', 'practice'), ('proctored', 'proctored'), ('timed', 'timed')], db_index=True, max_length=255),
        ),
        migrations.AlterField(
            model_name='historicalexam',
            name='exam_type',
            field=models.CharField(choices=[('onboarding', 'onboarding'), ('practice', 'practice'), ('proctored', 'proctored'), ('timed', 'timed')], db_index=True, max_length=255),
        ),
    ]
```

Could this be because we needed a migration file for these changes https://github.com/edx/edx-exams/pull/209?

Reply:
bah, that's 100% what's doing it. Thanks for catching it